src/main/resources/checkstyletask.properties

```diff
@@ -1 +0,0 @@
-checkstyle=com.puppycrawl.tools.checkstyle.CheckStyleTask
```

Comment:
From http://sourceware.org/ml/frysk/2008-q1/msg00004.html:
> git-mv does not do anything special, in fact.  GIT's support for file
renaming is just implicit detection, not explicit markers from user action.
So, what matters is doing a commit that removes file A and adds file B,
where A and B have completely identical contents (matching sha1sum).
To preserve history, you have to carefully do a commit that only renames,
and then a commit that touches the contents of the file.

So I need to make two commits